src/components/FeatureCards/FeatureCards.tsx

```diff
@@ -5,20 +5,53 @@ import { FeatureCard } from 'src/components';
 import { useFeatureCards } from 'src/hooks';
 import { useSettingsStore } from 'src/stores';
 import type { FeatureCardData } from 'src/types';
+import { deepMerge, removeEmpty } from 'src/utils';
 import { shallow } from 'zustand/shallow';
 import { FeatureCardsContainer } from '.';
 
 export const FeatureCards = () => {
   const [featureCards, setFeatureCards] = useState<FeatureCardData[]>([]);
-  const [disabledFeatureCards, welcomeScreenClosed] = useSettingsStore(
-    (state) => [state.disabledFeatureCards, state.welcomeScreenClosed],
-    shallow,
-  );
-
+  const [disabledFeatureCards, welcomeScreenClosed, languageMode] =
+    useSettingsStore(
+      (state) => [
+        state.disabledFeatureCards,
+        state.welcomeScreenClosed,
+        state.languageMode,
+      ],
+      shallow,
+    );
+  // get english response
   const { featureCards: data, isSuccess } = useFeatureCards();
+  // get translated response
+  const {
+    featureCards: translatedFeatureCards,
+    isSuccess: translatedFeatureCardsIsSuccess,
+  } = useFeatureCards({ translation: true });
+
+  // remove null´s (missing entries) from translated object
+  const cleanedTranslation = useMemo(() => {
+    const cleanedTranslation = translatedFeatureCards?.map((el) =>
+      removeEmpty(el),
+    );
+    return cleanedTranslation;
+  }, [translatedFeatureCards]);
+
+  // merge
+  const filteredResponse = data?.map((el) => {
+    const translatedItem = cleanedTranslation?.find(
+      (foundItem) =>
+        foundItem?.attributes?.DisplayConditions.id ===
+        el.attributes?.DisplayConditions.id,
+    );
+    if (translatedItem) {
+      return deepMerge(el, translatedItem);
+    }
+    return el;
+  });
+
```

Comment:
Should we put these in one `featureCardsFetched` useMemo?

Reply:
Thanks Eugene! Finally returned to that one you raised: I refactored it and put it into a single useMemo which seems better: But generally, I´d like to discuss this a little more with you

Reply:
Sure! Just let me know when it's best for you and my DM is always open 😄